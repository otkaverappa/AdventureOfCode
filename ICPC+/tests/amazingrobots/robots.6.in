5 8
########
##.#...#
##.###..
#....X.#
########
2
3 3 2 S
3 7 2 N
8 8
########
###....#
#...####
..###..#
#...#.##
##.X##.#
#.......
########
3
3 2 3 E
5 2 2 N
6 7 2 S
