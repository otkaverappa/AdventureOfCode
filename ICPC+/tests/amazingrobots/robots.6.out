8
E
S
E
E
N
E
E
E
